wordpress/Makefile

```diff
@@ -57,7 +57,6 @@ app/build:: .build/wordpress/deployer \
                            deployer/* \
                            manifest/* \
                            schema.yaml \
-                           .build/marketplace/deployer/envsubst \
```

Comment:
this has implication that we're no longer building this deployer from source but pulling it from GCR instead. I don't think we want to do that for the example repo, right?

Reply:
That's the implication, yes.

I have the impression that's desirable - we want partners to fork this repo, and we do not partners to build their own deployer images.

Let's discuss?

Reply:
Updated as per our discussion:

We should eventually pull the gcr.io/cloud-marketplace-tools/ images, but before then we should set up some form of CI/CD for the tools repo. Until then, consumers should rebuild.